benchmark/synthetic/models/lstm_text_generation_wikitext2.py

```diff
@@ -0,0 +1,190 @@
+'''
+Original Model from https://github.com/keras-team/keras/blob/master/examples/lstm_text_generation.py
+Benchmark for a LSTM model.
+Script template modified from TensorFlow Benchmark repo:
+https://github.com/tensorflow/benchmarks/blob/keras-benchmarks/scripts/keras_benchmarks/models/lstm_benchmark.py
+
+Used WikiText-2 character level dataset from
+https://einstein.ai/research/the-wikitext-long-term-dependency-language-modeling-dataset
+'''
+from __future__ import print_function
+import keras
+from keras.utils import multi_gpu_model
+
+from models import timehistory
+
+from keras.callbacks import LambdaCallback
+from keras.models import Sequential
+from keras.layers import Dense, Activation
+from keras.layers import LSTM
+from keras.optimizers import RMSprop
+import numpy as np
+import random
+import sys
+import io
+import re
+import os
+
+if keras.backend.backend() == 'tensorflow':
+    import tensorflow as tf
+if keras.backend.backend() == 'cntk':
```

Comment:
remove

Reply:
Done. Added a check for MXNet and TensorFlow backend. Now, the benchmark script only supports MXNet and TensorFlow as backend and fails for rest.
Thanks.